htdocs/search.php

```diff
@@ -1292,6 +1295,16 @@
             $sql_where[] = '`cache_status`.`allow_user_view`=1';
         }
 
+        //add selected caches to selected cachelist
+        if (isset($_REQUEST['addToList']) && isset($_REQUEST['addCache']) && isset($_REQUEST['selectCachelist'])) {
+            $list_caches_= $_REQUEST['addToList'];
+            $added_waypoints = addToList($list_caches);
+            $addCachelist= cachelist::getListById($_REQUEST['selectCachelist'] + 0); // null for invalid ID
```

Comment:
(int) $_REQUEST['selectCachelist'] anstatt + 0 machen bitte.

Reply:
Fixed it in new commit :).